javascript/javascript1/week3/javascript-warmup/When-will-we-be-there.js

```diff
@@ -0,0 +1,18 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+
+  function calcTravelTime(travelInformation){
+
+    const hoursWithDecimals = travelInformation.destinationDistance / travelInformation.speed
+    const hoursWithOutDecimals = Math.floor(hoursWithDecimals);
+    const minutesWithDecimals = (hoursWithDecimals - hoursWithOutDecimals)* 60 ;
+    const minutesWithOutDecimals = Math.round(minutesWithDecimals);
+    console.log(`${hoursWithOutDecimals} hours and ${minutesWithOutDecimals} minutes`);
+  }
+  
+  const travelTime = calcTravelTime(travelInformation);
```

Comment:
The variable declaration/assignment doesn't seem to do anything here and could be removed. 🙂